services/vision/builtin/tflite_detector.go

```diff
@@ -115,6 +115,9 @@ func addTFLiteModel(ctx context.Context, filepath string, numThreads *int) (*inf
 
 	model, err = loader.Load(filepath)
 	if err != nil {
+		if strings.Contains(err.Error(), "failed to load") {
+			return nil, errors.Wrapf(err, "file not found at %s", filepath)
```

Comment:
can you also find a way to "expand" this filepath to the full path? using function https://pkg.go.dev/path/filepath#Abs